modules/sc-mesh-secure-deployment/src/2_0/features/jamming/jamming_setup.py

```diff
@@ -163,34 +171,129 @@ def start_jamming_scripts(args, osf_ipv6_address) -> None:
         logger.info(f"Error starting jamming server/client scripts: {e}")
         raise Exception(e)
 
-def main():
+
+def get_device_identity(args) -> None:
+    """
+    Request device identity
+    """
+    # Store the current working directory
+    current_directory = os.getcwd()
+
+    try:
+        # Change the directory to the scripts_path
+        os.chdir(args.nats_scripts_path)
+
+        # Define the commands to execute
+        command = ["python", "_cli_command_get_identity.py"]
```

Comment:
These test scripts can change and not meant to be used like this.

Reply:
Ok, does this apply to client.py and config.py in nats/scripts as well?

Reply:
Yes,  nats/scripts are for testings the interface and those are same-time examples how to use the interface.

If someone decides to use the interfaces in there own code, similar functionality should done to own module as those testing tools can change (output which those are printing or even filenames can change).